packages/core/upload/admin/src/components/AssetTableList/utils/tableHeaders.js

```diff
@@ -0,0 +1,52 @@
+import { getTrad } from '../../../utils';
+
+export const tableHeaders = [
```

Comment:
I'm wondering if this should live inside of `constants.js` instead, because it is not really a utility.

Reply:
agreed

Reply:
sounds good indeed, felt strange to put that in a utils folder - thanks!